lms/static/js/wiki/CodeMirror.init.js

```diff
@@ -10,6 +10,11 @@ $(document).ready(function() {
   // Store the inital contents so we can compare for unsaved changes
     var initial_contents = editor.getValue();
 
+    // The Wiki associates a label with the text area that has ID id_content. However, when we swap in
+    // CodeMirror, that text area is hidden. We need to associate the label with CodeMirror (and there
+    // is JS code in the wiki that depends on "id_content" to get the content).
+    editor.getInputField().setAttribute('id', 'id_codemirror_content');
+    $(".control-label[for='id_content']")[0].setAttribute('for', 'id_codemirror_content');
```

Comment:
This is the content editor for the wiki page-- there can only be one. 

If there were multiple code mirrors, that would lead to a different bug (non-unique IDs).